src/Http/Routing/samples/MapActionSample/MapActionSample.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore" />
+    <Reference Include="Microsoft.AspNetCore.Diagnostics" />
+    <Reference Include="Microsoft.AspNetCore.Hosting" />
+    <!-- Mvc.Core is referenced only for its attributes -->
```

Comment:
Are these attributes going to be replaced with new ones?

Reply:
I don't think so. We've been trying to avoid duplicating attributes as that can lead to confusion.